### ISSUE TITLE
Removing redmine_acts_as_taggable_on

### ISSUE THREAD

**Issue by sdwolf** [ixti/redmine_tags#114]
Since a lot of confusion is created by needing to follow the instructions from [here](https://github.com/ixti/redmine_tags#to-make-it-actually-work) I've looked into what can be done to simplify the process.

I think there is a simpler solution than needing to dive into the criptic [redmine_acts_as_taggable_on](https://github.com/hdgarrood/redmine_acts_as_taggable_on) gem.

1. We simply copy the migration files from `rake acts_as_taggable_on_engine:install:migrations` into out `db/migrate` folder ([redmine_knowledgebase](https://github.com/alexbevi/redmine_knowledgebase) does this as well).
2. For each migration I add guard clauses to check if table/column/index exists (standard rails method calls), as expected if migrations were previously run.
3. When rolling back I will add a message that informs about tabels potentially being used by other plugins as well. This would be a prompt for contiune (Y/N). Also displaying (maybe even invoking directly) a grep command to search for such plugins.  
4. Since I'd like the prompt to appear only once, all these migrations will be merged into one.

What do you think of this approach?

**Comment by marius-balteanu** [comment]
I totally agree to move the required migrations from [redmine_acts_as_taggable_on](https://github.com/hdgarrood/redmine_acts_as_taggable_on) into this plugin. Most of the issues are generated by this.

For 3 we could also let the two tables (tags and taggings) as is and we just inform the user to manually remove the tables only if they are sure that are not other plugins that use the tables.

**Comment by ixti** [comment]
Agree with @marius-balteanu. No need for prompt. Just inform that they were left.
